crates/bevy_audio/src/audio_output.rs

```diff
@@ -72,8 +75,12 @@ where
     Source: Decodable,
 {
     let world = world.cell();
-    let audio_output = world.get_non_send::<AudioOutput<Source>>().unwrap();
-    let mut audio = world.get_resource_mut::<Audio<Source>>().unwrap();
+    let audio_output = world
+        .get_non_send::<AudioOutput<Source>>()
+        .expect("Could not find `AudioOutput` in the `World`.");
+    let mut audio = world
+        .get_resource_mut::<Audio<Source>>()
+        .expect("Could not find `Audio` in the `World`.");
```

Comment:
In both those cases, the issue would be either:
* the user is calling this system without adding the `AudioPlugin`
* the user is calling this system on a custom `Source` without having it set up first like in https://github.com/bevyengine/bevy/blob/0ccb9dd07e8c2c1663ad80272706aa3d2aa53cfd/crates/bevy_audio/src/lib.rs#L57-L59

It could be helpful to explain how to fix the issue

Reply:
Would this be a case where adding to the error codes is best for an explanation?